devstack/plugin.sh

```diff
@@ -35,8 +35,8 @@ function nfp_configure_neutron {
     iniset $NEUTRON_CONF admin_owned_resources_apic_tscp plumbing_resource_owner_user "neutron"
     iniset $NEUTRON_CONF admin_owned_resources_apic_tscp plumbing_resource_owner_password $ADMIN_PASSWORD
     iniset $NEUTRON_CONF admin_owned_resources_apic_tscp plumbing_resource_owner_tenant_name "service"
-    iniset $NEUTRON_CONF group_policy_implicit_policy default_ip_pool "11.0.0.0/8"
-    iniset $NEUTRON_CONF group_policy_implicit_policy default_proxy_ip_pool "192.169.0.0/16"
+    iniset $NEUTRON_CONF group_policy_implicit_policy default_ip_pool "172.16.0.0/12"
+    iniset $NEUTRON_CONF group_policy_implicit_policy default_proxy_ip_pool "192.168.0.0/16"
```

Comment:
OK to use the defaults defined in:
./neutron/services/grouppolicy/drivers/implicit_policy.py
./neutron/extensions/driver_proxy_group.py

The 2 lines are not required